pages/common/bzip2.md

```diff
@@ -14,3 +14,19 @@
 - Decompress a file to standard output:
 
 `bzip2 -dc {{path/to/compressed_file.bz2}}`
+
+- Test the integrity of each file inside the archive file:
+
+`bzip2 -t {{path/to/compressed_file.bz2}}`
```

Comment:
```suggestion
`bzip2 --test {{path/to/compressed_file.bz2}}`
```

Mandatory fix: [long option must be used if it's portable](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax).